teachertool/src/components/styling/ConfirmationModal.module.scss

```diff
@@ -0,0 +1,5 @@
+.confirmation-modal {
+    button.caution {
+        background-color: rgb(255, 71, 71);
```

Comment:
Can this color be derived from the toast error color? I think it will be the same color for all themes, so not a big issue to appear as a literal here. Though high-contrast might be a special case.

Reply:
I tried that at first, but the button looked disabled:
![image](https://github.com/microsoft/pxt/assets/69657545/10d1a754-8039-47ab-99f0-6a781171f8b0)


That said, the toast's accent color might work. I'll switch to that:
![image](https://github.com/microsoft/pxt/assets/69657545/88134785-5aad-4646-b6f0-62a96768cdf6)